setup.py

```diff
@@ -1,6 +1,8 @@
 # coding: utf-8
-
-from distutils.core import setup
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
 
 from skaero import __version__
```

Comment:
This can be a problem on installation, see https://packaging.python.org/guides/single-sourcing-package-version/ I usually follow approach 3 nowadays, just put a `version.py` under skaero.